circle.yml

```diff
@@ -2324,6 +2340,11 @@ linux-workflow: &linux-workflow
         percy: true
         requires:
           - build
+    - run-reporter-component-tests-chrome:
+        context: [test-runner:launchpad-tests, test-runner:percy]
```

Comment:
I'm not super familiar with this setup, but I think you could:
1. Use the `test-runner:cypress-record-key` context here rather than `test-runner:launchpad-tests`
2. Update `run-new-ui-tests` to parameterize the CYPRESS_RECORD_KEY variable it uses (currently hardcoded to `$TEST_LAUNCHPAD_RECORD_KEY`, you want `$MAIN_RECORD_KEY` with the cypress-record-key context)

That might get you passed the current errors and keep the existing `reporter-integration-tests` build recording as-is.